### PR TITLE
Add Jenkins Job Programmatically

### PR DESCRIPTION
**Pull request by ac-miller**
## what
* Add a job to Jenkins programmatically
* Add a webhook in GitLab to point at the Jenkins job

## why
* We needed a job in Jenkins
* We needed this job to be automatically created for students
* We needed this job to be automatically connected between GitLab and Jenkins

### CODE REVIEW

File: versions.tf [hunk 1]
```diff
@@ -1,10 +1,18 @@
 terraform {
-  required_version = ">= 0.13.0"
+  required_version = "0.13.5"
```
Comment (RothAndrew) [inline versions.tf]: The versions in this file shouldn't be pinned. They should stay open ended with a `>=` since this is the module, not the impl

**Review by RothAndrew — DISMISSED** [review]

**Review by Xerralt — DISMISSED** [review]

File: versions.tf (hunk 1 above)
Comment (ac-miller) [reply versions.tf]: This has been fixed.

**Review by Xerralt — APPROVED** [review]

**Pull request merged**